spec/rails_helper.rb

```diff
@@ -356,6 +353,9 @@ def synchronize(seconds = nil, errors: nil)
         .new(logging_prefs: { "browser" => browser_log_level, "driver" => "ALL" })
         .tap do |options|
           options.add_emulation(device_name: "iPhone 12 Pro")
+          options.add_argument(
+            '--user-agent="--user-agent="Mozilla/5.0 (Windows Phone 10.0; Android 4.2.1; Microsoft; Lumia 640 XL LTE) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Mobile Safari/537.36 Edge/12.10166"',
+          )
```

Comment:
Do we need both an Android and an iOS agent here? Since device is iPhone, maybe we stick with just that user agent?

Reply:
Pushed another commit -  I copy/pasted the wrong user agent. I've changed it to the iOS agent, which is what we are emulating.